docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -478,10 +478,44 @@ Limitations:
 5.0 Model run requests
 ++++++++++++++++++++++
 
-.. todo::
+.. list-table:: Model run plan as of October 4, 2024
+  :header-rows: 1
 
-  Fill in this section as we continue to work
+  * - Number
+    - Run
+    - Status
+    - Priority
+    - Number of draws
+    - Population size per draw
+    - Note
+  * - 1
+    - Wave I Pakistan maternal V&V
+    - Incomplete
+    - N/A
+    - 20
+    - 20,000
```

Comment:
I have always found this confusing - we usually give the engineers a pop size and then they determine the spread over seeds. For example, if we say 100,000 population they can do a run size of 10,000 over 10 seeds or 5,000 pop size with 20 seeds. However, since the RT is now maintaining the model spec files and helping with kicking off runs, maybe that approach isn't appropriate any more. Something we can discuss further later! 

At any rate, I would have this be 10,000 sims at 10 seeds (total pop is then 100,000) for a "standard" V&V approach. Usually we list that as "100,000" in this column and then in the notes include something about 10 seeds. 

The NO docs are confusing but they are doing 20,000 per seed in production runs, so similar to draws we are cutting down the size.

Reply:
Okay, thank you for explaining! I'll summarize this in the note.